### DISCUSSION TITLE
Within a table, how can I have a merged cell?

### DISCUSSION THREAD

**Discussion by LizThompson93** [squidfunk/mkdocs-material#7100] [Q&A] [answered]
I am looking to achieve something like this using [data tables](https://squidfunk.github.io/mkdocs-material/reference/data-tables/#sortable-tables-mkdocsyml).
![image](https://github.com/squidfunk/mkdocs-material/assets/96261305/952830c7-51ff-481c-8bb9-17fadb3870fd)

Where "field_3" has two columns beneath it. I have tested it with regular syntax but it does not render correctly.

```
| **User Management API**                                    |
|------------------------------------------------------------|
| Endpoint      | Method | Description                       |
| ------------- | ------ | ----------------------------------|
| `/users`      | `GET`  | material-check Fetch all users    |
| `/users`      | `POST` | material-check-all Create a user  |
| `/users/{id}` | `GET`  | material-check Fetch user by ID   |
| `/users/{id}` | `PUT`  | material-check-all Update user    |
| `/users/{id}` | `DELETE`| material-close Delete user       |
```

![image](https://github.com/squidfunk/mkdocs-material/assets/96261305/8de671a6-a6c2-44dc-8735-4f0d4e2f3471)

I have also attempted to use read_excel() using mkdocs-table-reader-plugin to no avail:
![image](https://github.com/squidfunk/mkdocs-material/assets/96261305/e0143e9e-53fb-4d1d-ae17-ce569b8a31bc)

What are recommendations, extensions, or plugins that may help? I tried using embedded iframes but it had mobile issues that I'm not too fond of.

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. However, this is an exact duplicate of https://github.com/squidfunk/mkdocs-material/discussions/4845.
